src/DotNetBridge/Bridge.cs

```diff
@@ -445,4 +450,22 @@ internal static byte[] StringToNullTerminatedBytes(string str)
             return Encoding.UTF8.GetBytes(str + Char.MinValue);
         }
     }
+
+    internal static class SchemaManipulation2
+    {
+        [TlcModule.EntryPoint(Name = "Transforms.ColumnConcatenatorV2", Desc = ColumnConcatenatingTransformer.Summary, UserName = ColumnConcatenatingTransformer.UserName, ShortName = ColumnConcatenatingTransformer.LoadName)]
```

Comment:
Should this be moved in to SchemaManipulation.cs or something like that?

Reply:
done

---
In reply to: [326341385](https://github.com/microsoft/NimbusML/pull/270#discussion_r326341385) [](ancestors = 326341385)